clusterloader2/pkg/measurement/common/network-policy/manifests/dep-test-client-policy-creation.yaml

```diff
@@ -34,7 +34,8 @@ spec:
           name: npdelaymetrics
           protocol: TCP
         imagePullPolicy: Always
-        image: gcr.io/k8s-staging-perf-tests/network-policy-enforcement-latency/policy-creation-enforcement-latency:v0.0.1
+        # image: gcr.io/k8s-staging-perf-tests/network-policy-enforcement-latency/policy-creation-enforcement-latency:v0.0.1
+        image: docker.io/sanamsarath/policy-creation-enforcement-latency:v0.0.1
```

Comment:
why are we using custom image here? If that's needed to add new commands, then we should publish it to ghcr and use image from there instead of docker.io